### PR TITLE
Clean sync tables on App Key change

### PR DESCRIPTION
**Pull request by manaftryzens**

### CODE REVIEW

File: etc/cron_groups.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Cron:etc/cron_groups.xsd">
+    <group id="yotpo_smsbump_customers_sync">
```
Comment (EranAvidor) [inline etc/cron_groups.xml:3]: @manaftryzens please rename the group.
```suggestion
    <group id="yotpo_messaging_customers_sync">
```

File: Observer/Config/CronFrequency.php [hunk 2]
```diff
@@ -29,7 +29,7 @@ class CronFrequency extends YotpoCoreCronFrequency
      * @var array <mixed>
      */
     protected $cronFrequency = [
-        'customers_sync_frequency' => ['config_path' => '','job_code' => 'yotpo_cron_smsbumb_customers_sync']
```
Comment (EranAvidor) [inline Observer/Config/CronFrequency.php:32]: @manaftryzens Is this a fix or just a typo (together with https://github.com/YotpoLtd/magento2-module-messaging/pull/72/files#diff-044dd0540d0b1a26871b5f18bfcbf3a1684560b35f94a0567918c7f97cb6eed1L24)

**Review by EranAvidor — CHANGES_REQUESTED** [review]

File: etc/cron_groups.xml (hunk 1 above)
Comment (manaftryzens) [reply etc/cron_groups.xml:3]: This is done

File: Observer/Config/CronFrequency.php (hunk 2 above)
Comment (manaftryzens) [reply Observer/Config/CronFrequency.php:32]: Yes, It is a typo correction.

File: etc/cron_groups.xml (hunk 1 above)
Comment (manaftryzens) [reply etc/cron_groups.xml:3]: This is done

**Pull request merged**